hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestSplitTableRegionProcedure.java

```diff
@@ -114,39 +111,7 @@ private static void setupConf(Configuration conf) {
     conf.setInt(MasterProcedureConstants.MASTER_PROCEDURE_THREADS, 1);
     conf.setLong(HConstants.MAJOR_COMPACTION_PERIOD, 0);
     conf.set("hbase.coprocessor.region.classes",
-      RegionServerHostingReplicaSlowOpenCopro.class.getName());
-    conf.setInt("hbase.client.sync.wait.timeout.msec", 1500);
-  }
-
-  /**
-   * This copro is used to slow down opening of the replica regions.
-   */
-  public static class RegionServerHostingReplicaSlowOpenCopro
```

Comment:
I've moved this up to a standalone class

Reply:
👍